.pre-commit-config.yaml

```diff
@@ -24,6 +24,7 @@ repos:
   rev: 5.9.3
   hooks:
   - id: isort
+    args: ["--profile", "black"]
```

Comment:
Hi, this should be already addressed by the `.isort.cfg` file here: https://github.com/fastmachinelearning/qonnx/blob/main/.isort.cfg#L11

Reply:
Thanks for pointing that out, that line is now removed.